src/main/java/com/cs50/FinanceApp/services/SellServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package com.cs50.FinanceApp.services;
+
+import com.cs50.FinanceApp.controllers.SellRequest;
+import com.cs50.FinanceApp.model.Purchases;
+import com.cs50.FinanceApp.model.TransactionType;
+import com.cs50.FinanceApp.model.User;
+import com.cs50.FinanceApp.repositories.PurchasesRepository;
+import com.cs50.FinanceApp.repositories.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class SellServiceTest {
```

Comment:
Ogólnie to powinno być w innym miejscu, nie `src/main/java` tylko chyba `src/test/...` . Ale muszę sobie to odpalić i sprawdzić